client/state/billing-transactions/reducer.js

```diff
@@ -76,4 +77,5 @@ export default combineReducers( {
 	items,
 	requesting,
 	sendingReceiptEmail,
+	individualTransactions,
```

Comment:
Question: is there any possibility for an individual transaction to also be included in the list of transactions (the `items` state tree)? If there is any chance for that to happen, should we also be updating the `items` state tree everytime we update the `individialTransactions` tree? And if we do that, should we use the `items` tree instead for individual transactions in order to maintain consistence and avoid data duplication and redundancy?

Reply:
I get your point, and it's perfectly a valid one.

Perhaps add a `// TODO` comment somewhere so we make sure we know about this, and hopefully fix it at some point?